ls1intum/ArTEMiS
# [WIP] Bugfix/fontawesome icon not found in asssessment result view

### Checklist

### Description
There were no checkmark icons in the Assessment Result View for the student that represent the points for each element. 

### Steps for Testing
1. Log in to ArTEMiS as a student
2. Navigate to the assessment of a submission 
3. Check, if the icons are displayed.
